modules/kml/src/kml-loader.js

```diff
@@ -25,22 +22,16 @@ function parseTextSync(text, options) {
   options.kml = options.kml || {};
   options.gis = options.gis || {};
 
-  const xml = XMLLoader.parseTextSync(text);
-  const kmlLoader = new KMLParser();
-  let kml = kmlLoader.parse(xml, options);
-
-  if (options.kml.normalize) {
-    kml = normalizeKML(kml);
-  }
+  const doc = new DOMParser().parseFromString(text, 'text/xml');
+  const geojson = kml(doc);
 
   switch (options.gis.format) {
     case 'geojson':
-      return convertKMLToGeoJson(kml);
+      return geojson;
     case 'binary':
-      const geojson = convertKMLToGeoJson(kml);
       return geojsonToBinary(geojson.features);
     default:
-      return kml;
+      return doc;
```

Comment:
- Do we want to offer the 'raw' option? It seems hard to keep stable...
- At least should we throw on default and add a case for `raw`?

Reply:
I think `raw` is equivalent to an `XMLLoader`... it just passes the text to `DOMParser` and that's it. I added a separate case for `raw` for now.